macOSUpgrade.sh

```diff
@@ -354,46 +359,17 @@ fi
 </plist>
 EOP
 
-##Set the permission on the file just made.
-/usr/sbin/chown root:wheel "$osinstallersetupdAgentSettingsFilePath"
-/bin/chmod 644 "$osinstallersetupdAgentSettingsFilePath"
+    ##Set the permission on the file just made.
+    /usr/sbin/chown root:wheel "$osinstallersetupdAgentSettingsFilePath"
+    /bin/chmod 644 "$osinstallersetupdAgentSettingsFilePath"
+
+fi
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # APPLICATION
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-if [[ ${pwrStatus} == "OK" ]] && [[ ${spaceStatus} == "OK" ]]; then
-    ##Launch jamfHelper
-    if [ ${userDialog} -eq 0 ]; then
-        /bin/echo "Launching jamfHelper as FullScreen..."
-        /Library/Application\ Support/JAMF/bin/jamfHelper.app/Contents/MacOS/jamfHelper -windowType fs -title "" -icon "$icon" -heading "$heading" -description "$description" &
-        jamfHelperPID=$!
-    else
-        /bin/echo "Launching jamfHelper as Utility Window..."
-        /Library/Application\ Support/JAMF/bin/jamfHelper.app/Contents/MacOS/jamfHelper -windowType utility -title "$title" -icon "$icon" -heading "$heading" -description "$description" -iconSize 100 &
-        jamfHelperPID=$!
-    fi
-    ##Load LaunchAgent
-    if [[ ${fvStatus} == "FileVault is On." ]] && [[ ${currentUser} != "root" ]]; then
-        userID=$( /usr/bin/id -u "${currentUser}" )
-        /bin/launchctl bootstrap gui/"${userID}" "$osinstallersetupdAgentSettingsFilePath"
-    fi
-    ##Begin Upgrade
-    /bin/echo "Launching startosinstall..."
-    ##Check if eraseInstall is Enabled
-    if [[ $eraseInstall == 1 ]]; then
-        eraseopt='--eraseinstall'
-        /bin/echo "   Script is configured for Erase and Install of macOS."
-    fi
-
-    osinstallLogfile="/var/log/startosinstall.log"
-    if [ "$versionMajor" -ge 14 ]; then
-        eval /usr/bin/nohup "\"$OSInstaller/Contents/Resources/startosinstall\"" "$eraseopt" --agreetolicense --nointeraction --pidtosignal "$jamfHelperPID" >> "$osinstallLogfile" &
-    else
-        eval /usr/bin/nohup "\"$OSInstaller/Contents/Resources/startosinstall\"" "$eraseopt" --applicationpath "\"$OSInstaller\"" --agreetolicense --nointeraction --pidtosignal "$jamfHelperPID" >> "$osinstallLogfile" &
-    fi
-    /bin/sleep 3
```

Comment:
Thank you for your consideration.

I forgot to remove these lines.

Reply:
@taniguti 

If you can because we want to concentrate on the https://github.com/kc9wwh/macOSUpgrade/pull/97#discussion_r266288181, can you split this PR?

Reply:
got it.
I'll make another PR for it.